tests/ops/integration_tests/saas/test_gorgias_task.py

```diff
@@ -0,0 +1,343 @@
+        policy,
+        graph,
+        [gorgias_connection_config],
+        {"email": gorgias_identity_email},
+        db,
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:customer"],
+        min_size=1,
+        keys=[
+            "id",
+            "external_id",
+            "active",
+            "name",
+            "email",
+            "firstname",
+            "lastname",
+            "language",
+            "timezone",
+            "created_datetime",
+            "updated_datetime",
+            "meta",
+            "data",
+            "note",
+        ],
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:search"],
+        min_size=1,
+        keys=[
+            "data",
+            "object",
+            "uri",
+            "meta",
+        ],
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:tickets"],
+        min_size=1,
+        keys=[            
+            "id",
+            "uri",
+            "external_id",
+            "language",
+            "status",
+            "priority",
+            "channel",
+            "via",
+            "from_agent",
+            "customer",
+            "assignee_user",
+            "assignee_team",
+            "subject",
+            "meta",
+            "tags",
+            "is_unread",
+            "spam",
+            "created_datetime",
+            "opened_datetime",
+            "last_received_message_datetime",
+            "last_message_datetime",
+            "updated_datetime",
+            "closed_datetime",
+            "snooze_datetime",
+            "trashed_datetime",
+            "integrations",
+            "messages_count",
+            "excerpt",
+        ],
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:ticket_message"],
+        min_size=1,
+        keys=[
+            "id",
+            "uri",
+            "message_id",
+            "ticket_id",
+            "external_id",
+            "public",
+            "channel",
+            "via",
+            "source",
+            "sender",
+            "integration_id",
+            "intents",
+            "rule_id",
+            "from_agent",
+            "subject",
+            "body_text",
+            "body_html",
+            "stripped_text",
+            "stripped_html",
+            "stripped_signature",
+            "headers",
+            "attachments",
+            "actions",
+            "macros",
+            "meta",
+            "created_datetime",
+            "sent_datetime",
+            "failed_datetime",
+            "deleted_datetime",
+            "opened_datetime",
+            "last_sending_error",
+            "is_retriable",
+        ],
+    )
+
+    # verify we only returned data for our identity email
+    assert v[f"{dataset_name}:customer"][0]["email"] == gorgias_identity_email
+    user_id = v[f"{dataset_name}:customer"][0]["id"]
+
+    # assert v[f"{dataset_name}:user_identities"][0]["value"] == gorgias_identity_email
```

Comment:
Don't leave any commented code, you can uncomment this since we still need to check the identities have the correct value

Reply:
The above commented code no needed.so,we removed